.github/workflows/lint-bpf-checks.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
         uses: dorny/paths-filter@b2feaf19c27470162a626bd6fa8438ae5b263721
         id: changes
         with:
+          base: ${{ github.event.pull_request.base.sha || github.event.before }}
```

Comment:
Where is this coming from? Could it be something we removed for a legitimate reason in master and forgot to backport to v1.10?

Reply:
This line was removed in https://github.com/cilium/cilium/commit/652da822debe3404ed0b287aa18e7d30d8c17783#diff-2540bd4e18ec73cb134ede1c7ce5126ce3f6f69cb391b91bf4506dc5d3548f1a, which we didn't backport to v1.10. We should remove this line and the other two in `.github/workflows/lint-build-commits.yaml`.

Reply:
Come to think of it, we should just apply https://github.com/cilium/cilium/pull/16311 again on this branch and squash the three commits I think.

Reply:
I've used the v1.10 as reference. It's coming from [here](https://github.com/cilium/cilium/blob/v1.10/.github/workflows/lint-bpf-checks.yaml#L33)

Reply:
Yeah, I know. But you only did that because you don't want to reimplement all changes made when we branched to v1.10, right? So we want the workflows to be the same as master, with just the changes required in stable branches (e.g., removing cron triggers, changing trigger phrases, etc.). This isn't one of those changes; those lines were removed in master to shut a warning. Thus, I think we should remove them in v1.11 as well.